README.md

```diff
@@ -39,100 +39,71 @@ time when the event is inserted by the client into a remote session. In this cas
+Review and modify the server code based on the sample project in the sample-hacep-project/sample-hacep-project-kjar directory. 
+The complex event processing logic is defined by the DRL rules in the src/main/resources/org.drools.cep subdirectory.
+- 2) Build the project using the standard Maven command: 
 
-### Build the pods
 ```sh
 mvn clean install -DskipTests
 ```
-### Deployment
-Are available three modules 
-
-- Springboot     ( openshift-kie-springboot.jar )
-- Thorntail      ( openshift-kie-thorntail.jar )
-- Jdk HttpServer ( openshift-kie-jdkhttp.jar )
-
-choose your and move in the respective module to run the resepcitve command 
-to create the Container image and then deploy on Openshift, as described in the module's README.md
-
-
-### Client outside cluster
-
-If you plan to use a client outside openshift
-you need to expose kafka with a route
-in the kafka cluster creation you could enable the https endpoint with
-listener external of type route
-
-```json
-apiVersion: kafka.strimzi.io/v1beta1
-kind: Kafka
-metadata:
-  name: my-cluster
-spec:
-  kafka:
-    version: 2.2.1
-    replicas: 3
-    listeners:
-      plain: {}
-      external:
-        type: route
-```
+- 3)
+Use the OpenShift operator infrastructure to install [Red Hat AMQ Streams](https://access.redhat.com/documentation/en-us/red_hat_amq/7.3/html/using_amq_streams_on_openshift_container_platform/index) .
 
-### Client module
-- sample-hacep-project-client 
+- 4)
+Using the KafkaTopic resource on Red Hat OpenShift Container Platform, create the topics from all the YAML files in the kafka-topics subdirectory. 
+For instructions about creating topics using the KafkaTopic resource, see [Using the topic operator](https://access.redhat.com/documentation/en-us/red_hat_amq/7.4/html/using_amq_streams_on_openshift_container_platform/using-the-topic-operator-str) in the Red Hat AMQ documentation.
 
-#### Client configuration
-From the root of the client module:
-Generate a keystore and use "password" as a password
+- 5)
+In order to enable application access to the ConfigMap that is used in the leader election, you must configure role-based access control. 
+Change to the springboot directory and enter the following commands: 
 ```sh
-keytool -genkeypair -keyalg RSA -keystore src/main/resources/keystore.jks
-```
-extract the cert from openshift with:
-```sh
-oc extract secret/my-cluster-cluster-ca-cert --keys=ca.crt --to=- > src/main/resources/ca.crt
+oc create -f kubernetes/service-account.yaml
+oc create -f kubernetes/role.yaml
+oc create -f kubernetes/role-binding.yaml
 ```
+  
+  For more information about configuring role-based access control in Red Hat OpenShift Container Platform, see [Using RBAC to define and apply permissions](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.1/html/authentication/using-rbac) in the Red Hat OpenShift Container Platform product documentation.
```

Comment:
Please use a link to community docs here: https://docs.okd.io/latest/admin_guide/manage_rbac.html

Reply:
Ok

Reply:
Ok